gcp_variant_transforms/testing/integration/run_tests.py

```diff
@@ -148,28 +146,52 @@ def _handle_failure(self, response):
             'No traceback. See logs for more information on error.')
 
   def validate_table(self):
-    """Runs a simple query against the output table and verifies aggregates."""
+    """Runs queries against the output table and verifies aggregates."""
     client = bigquery.Client(project=self._project)
-    # TODO(bashir2): Create macros for common queries and add the option for
-    # having a list of queries instead of just one.
-    query = self._validation_query.format(TABLE_NAME=self._table_name)
-    query_job = client.query(query)
+    # TODO(bashir2): Create macros for common queries
+    query_formatter = QueryFormatter(self._table_name)
+    for assertion_config in self._assertion_configs:
+      query = query_formatter.format_query(assertion_config['query'])
+      assertion = Assertion(client, query, assertion_config['expected_result'])
+      assertion.run_assertion()
+
+
+class Assertion(object):
+  """Runs a simple query against the output table and verifies aggregates."""
```

Comment:
nit: please rephrase the comment to something like "Runs a query and verifies that the output matches the expected result".